[DOCS] Add replication lag script (#4620)
yugabyte/yugabyte-db: docs/content/latest/deploy/multi-dc/2dc-deployment.md

**For bidirectional replication**

Repeat the steps above, but pump data into “yugabyte-consumer”. To avoid primary key conflict errors, keep the key space for the two universes separate.

> {{< note title="How to check replication lag" >}}

Reviewer: ```suggestion
{{< note title="Note" >}}
```
The Note title is only intended for Note, Important, Warning, and other admonitions.

Author: @stevebang  On the same page above it's a "tip" with a custom title `Recommended Reading`.

Reviewer: OK, letting it slide for now — this use and others should be handled differently. But, I will have to propose an alternative.

> Replication lag is computed at the tablet level by subtracting the `last_read_hybrid_time` from the `hybrid_clock_time`. 

Reviewer: Change to:

Replication lag is computed at the tablet level as:

```
 replication lag = hybrid_clock_time - last_read_hybrid_time
```
- `hybrid_clock_time`: <add definition or links>
- `last_read_hybrid_time`: <add definition or links>

> [Here](/files/determine_replication_lag.sh) is an example script that does it for you. 

Reviewer: ```suggestion
An example script [`determin_replication_lag.sh`](/files/determine_replication_lag.sh) calculates replication lag for you. 
```

> The script requires [`jq`](https://stedolan.github.io/jq/) package.

Reviewer: ```suggestion
The script requires the [`jq`](https://stedolan.github.io/jq/) package.
```

> The example below would give you a summary of all tables on a given cluster. You can also request an individual table:

Reviewer: ```suggestion
In the example below, a replication lag summary is generated for all tables on a cluster. You can also request an individual table:
```
